v22.1/take-and-restore-encrypted-backups.md

```diff
@@ -161,6 +161,8 @@ For example, the encrypted backup created in the previous example can be restore
 (1 row)
 ~~~
 
+To restore from a specific backup, use [`RESTORE FROM {subdirectory} IN ...`](restore.html#restore-a-specific-backup).
```

Comment:
nit: remove "specific"? a user may get the wrong idea that they can restore a single incremental backup or something.